design/Implemented/priority-class-name-support_design.md

```diff
@@ -582,6 +624,102 @@ With the introduction of VGDP micro-services (as described in the VGDP micro-ser
 
 This ensures that all pods created by Velero for data movement operations (CSI snapshot data movement, PVB, and PVR) use a consistent approach for priority class name configuration through the node-agent-configmap.
 
+## ConfigMap Update Strategy
```

Comment:
This change looks out of the scope of the current PR.

As the node-agent config design, the configurations are loaded for once during node-agent start/restart.
At present, we don't see this requirement, but if we want to enhance it for dynamic load of node-agent configs, we should add design modification based on the node-agent config design first.